explainer.md

```diff
@@ -22,118 +71,260 @@ place associated with the application (such as the shelf or home screen).
-  system shelf area). We could also explore icon badging on the drive-by web.
-  This naturally leads into...
-* Providing per-tab and per-window badging. The current proposal is for a global badge
-  for the application. See [#1](https://github.com/WICG/badging/issues/1).
-* Support apps that want to render a small status indicator (e.g., a music app shows ▶️	
-  or ⏸️; a weather app shows ⛈️ or ⛅️).
-* Setting the badge from a service worker (e.g. an email app updating an unread count).
+* Support rendering a small status indicator (e.g., a music app shows ▶️ or ⏸️; a weather app shows ⛈️ or ⛅️).
+* Support setting the badge from a service worker (e.g. an email app updating an unread count in the background).
+* Support badges on bookmark icons (this is probably up to user agents to support, and may not need anything new API wise).
+* Support for non-inherited badges (see [Issue 42](https://github.com/WICG/badging/issues/42)).
+* Support for unicode glyph badges.
 
 Examples of sites that may use this API:
 
-* Chat, email and social apps, to signal that new messages have arrived.
-* Productivity apps, to signal that a long-running background task (such as
-  rendering an image or video) has completed.
-* Games, to signal that a player action is required (e.g., in Chess, when it is
-  the player's turn).
+* Chat, email and social apps; to signal that new messages have arrived.
+* Productivity apps, to signal that a long-running background task (such as rendering an image or video) has completed.
+* Games, to signal that player action is required (e.g., in Chess, when it is the player's turn).
 
 Advantages of using the badging API over notifications:
 
-* Can be used for much higher frequency events than notifications, because each
-  new event does not disrupt the user.
-* There is no need to request permission to use the badging API, since it is
-  much less invasive than a notification.
+* Can be used for much higher frequency events than notifications, because each new event does not disrupt the user.
+* There is no need to request permission to use the badging API, since it is much less invasive than a notification.
 
-(Typically, sites will want to use both APIs together: notifications for
-high-importance events such as new direct messages or incoming calls, and badges
-for all new messages including group chats not directly addressed to the user.)
+(Typically, sites will want to use both APIs together: notifications for high-importance events such as new direct messages or incoming calls, and badges for all new messages including group chats not directly addressed to the user.)
 
 ## API proposal
 
 ### The model
 
-There is a single global badge associated with each Web
-application (as defined in [Web app
-manifest](https://www.w3.org/TR/appmanifest/)). At any time, the badge is set with:
+1. A Badge is associated with a [scope](https://www.w3.org/TR/appmanifest/#navigation-scope).
```

Comment:
```suggestion
1. If the web app is installed, a Badge is associated with a [scope](https://www.w3.org/TR/appmanifest/#navigation-scope).
```

Reply:
I think this change is incorrect? All Badges are tied to a scope, web applications don't really come into it. I could link to service worker [registration scope](https://w3c.github.io/ServiceWorker/#service-worker-registration-scope) here, instead of the appmanifest scope if it makes things more clear?

We might want to move `scope` into its own spec, and reference it from ServiceWorker, AppManifest and Badging, seeing as we seem to be reusing it a bit.

Reply:
Yeah, just didn't want it to read like "this is tied to web manifest". Step one makes it seem like it's really tied to it.

Reply:
Cool, I've done that, and I've added a note that we may want to extract it into its own spec.